ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -366,7 +366,7 @@ def set_action_for_agent(
             return
         action_spec = self._env_specs[behavior_name].action_spec
         num_agents = len(self._env_state[behavior_name][0])
-        action = action_spec._validate_action(action, None, behavior_name)
+        action = action_spec._validate_action(action, 1, behavior_name)
         if behavior_name not in self._env_actions:
             self._env_actions[behavior_name] = action_spec.empty_action(num_agents)
```

Comment:
can we set this to 1?

Reply:
What do you mean?/ You mean by default?

Reply:
oh I remember, all other actions are zero'd except the action being set for the agent.